.clang-format

```diff
@@ -0,0 +1,23 @@
+BasedOnStyle: LLVM
+BraceWrapping:
+  SplitEmptyFunction: false
+  SplitEmptyRecord: false
+IndentWidth: 3
+ColumnLimit: 120
+PointerAlignment: Left
+AccessModifierOffset: -3
+AlwaysBreakTemplateDeclarations: Yes
+AlignArrayOfStructures: Left
+AlignConsecutiveAssignments: Consecutive
+AlignConsecutiveDeclarations: Consecutive
+AlignEscapedNewlines: Left
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: Empty
+AllowShortFunctionsOnASingleLine: Inline
+BreakBeforeBraces: Custom
+BreakConstructorInitializers: BeforeComma
+BreakInheritanceList: BeforeComma
+ConstructorInitializerIndentWidth: 3
+ContinuationIndentWidth: 3
+MaxEmptyLinesToKeep: 3
+SortIncludes:    Never
```

Comment:
At this point, I wish we had this in the repo. However, should come in a separate PR.

Reply:
Oh, thanks, I must have added it by mistake.